tutor/src/screens/assignment-edit/tasking.js

```diff
@@ -242,6 +219,7 @@ class Tasking extends React.Component {
               name={`tasking_plans[${index}].closes_at`}
               onChange={this.onClosesChange}
               disabledDate={this.course.isInvalidAssignmentDate}
+              course={this.course}
```

Comment:
Maybe just pass the function `momentInZone` instead of the whole course?

Reply:
That's a great idea, I'm going to update that and fix up a different issue I found.